Packs/CommonScripts/ReleaseNotes/1_11_35.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: SetDateField
+
+- Sets a custom incident field with current date (Available from Cortex XSOAR 5.0.0).
```

Comment:
It's not a new integration, just updating it to use Py3. Please change the RN to match the changes.